packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -93,17 +93,20 @@ class ExpansionPanel {
 ///
 ///  * [ExpansionPanel]
 ///  * <https://material.google.com/components/expansion-panels.html>
-class ExpansionPanelList extends StatelessWidget {
-  /// Creates an expansion panel list widget. The [expansionCallback] is
-  /// triggered when an expansion panel expand/collapse button is pushed.
+
+class ExpansionPanelList extends StatefulWidget {
+
+  ///Expansion Panel List
   const ExpansionPanelList({
     Key key,
     this.children = const <ExpansionPanel>[],
     this.expansionCallback,
-    this.animationDuration = kThemeAnimationDuration
-  }) : assert(children != null),
-       assert(animationDuration != null),
-       super(key: key);
+    this.animationDuration = kThemeAnimationDuration,
+    this.allowMultiplePanelsOpen = true
+  })
+      : assert(children != null),
+        assert(animationDuration != null),
```

Comment:
should probably assert `allowMultiplePanelsOpen` is non-null too, unless you support null.
And document that it must not be null in the class constructor docs.